src/browser/api/application.js

```diff
@@ -278,7 +278,14 @@ Application.close = function(identity, force, callback) {
         Window.close(mainWindowIdentity, force, callback);
     }
 };
-
+Application.destroy = function(identity, cb, nack) {
```

Comment:
if this is an ack maybe lets rename `cb` to `ack`. Also might want to just `return` or `throw` here if this is backing an API call to be consistent with the new pattern.